Mission involves creep.lua

```diff
@@ -0,0 +1,86 @@
+local argsUtil = require("Module:ArgsUtil")
```

Comment:
nitpick: please, `args_util` - that's how I've been calling these since like 2018 and itll make it a lot easier for me to help you 😅